crates/miden-proving-service-client/Cargo.toml

```diff
@@ -17,7 +17,8 @@ crate-type = ["lib"]
 default = ["std"]
 std = ["miden-objects/std", "miden-tx/std"]
 tx-prover = ["miden-tx/async"]
-batch-prover = ["miden-tx/async", "dep:tokio"]
+batch-prover = ["dep:miden-tx", "dep:tokio"]
+block-prover = ["dep:miden-tx", "dep:tokio"]
```

Comment:
Why do `batch-prover` and `block-prover` features need `dep:miden-tx`?

Reply:
It's not needed. I just removed it.